crates/forge/bin/cmd/build.rs

```diff
@@ -18,6 +18,9 @@ use watchexec::config::{InitConfig, RuntimeConfig};
 
 foundry_config::merge_impl_figment_convert!(BuildArgs, args);
 
+// Extensions accepted by `forge build`
+const BUILD_EXTENSIONS: &[&str] = &["sol", "yul", "vy", "vyi"];
```

Comment:
can we use `MultiCompilerLanguage::EXTENSIONS` here? It would require including `Language` trait from compilers

Reply:
yep, changed in https://github.com/foundry-rs/foundry/pull/8149/commits/0e71ca5e9c2e12b37696a9261cc7bda999076918